README.md

```diff
@@ -32,6 +32,60 @@ environment.
     `npm start`
 
 
+### Setup Google Cloud Persistance
+
+6. Install Google Cloud module:
+
+    `npm install gcloud --save-dev`
+
+7. Install Google Cloud Datastore module:
+
+    `npm install @google-cloud/datastore --save-dev`
+
+7. [Follow instructions](https://cloud.google.com/datastore/docs/) for setting up Google Cloud Datastore. After creating a project and creating two entities (Collector, Selector), you can move on.
```

Comment:
I'm not sure it is necessary to create two entities at this step. Also, I think follow instructions on _this link_ is too generic. I can follow it because of using gcloud previously, but there should be more step by step instructions. I do not think there is that much involved, but listing "create a new project" is more important that pointing to google documentation.